extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -101,7 +101,7 @@ static public function onTitleGetSquidURLs( Title $title, Array &$urls ) {
 
 			// Mercury API call from Hapi.js to MediaWiki e.g.
 			// http://elderscrolls.wikia.com/wikia.php?controller=MercuryApi&method=getArticle&title=Morrowind
-			$urls[] = MercuryApiController::getUrl( 'getArticle', [ 'title' => $title->getPartialURL() ] );
+			$urls[] = MercuryApiController::getUrl( 'getPage', [ 'title' => $title->getPartialURL() ] );
```

Comment:
We should keep both urls here for now and create a ticket to clean it up after `getArticle` is no longer used.